pinot-core/src/main/java/org/apache/pinot/core/data/readers/RecordReaderUtils.java

```diff
@@ -112,8 +112,9 @@ public static Object convert(FieldSpec fieldSpec, @Nullable Object value) {
   public static Object convertSingleValue(FieldSpec fieldSpec, @Nullable Object value) {
     if (value == null) {
       // Do not allow default value for time column
```

Comment:
Move this comment or add a description to `NullValueTransformer`.

Reply:
Done.